spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+describe SubmissionBuilder::Ty2024::States::Nj::Documents::Nj1040, required_schema: "nj" do
+  describe ".document" do
+    let(:intake) { create(:state_file_nj_intake, filing_status: "single", municipality_code: "0101") }
+    let(:submission) { create(:efile_submission, data_source: intake) }
+    let(:build_response) { described_class.build(submission, validate: false) }
+    let(:xml) { Nokogiri::XML::Document.parse(build_response.document.to_xml) }
+
+    context "single filer" do
+      it "correctly fills answers" do
+        expect(xml.document.at('FilingStatus Single').text).to eq "X"
```

Comment:
[pebble] 
- similar feedback to before, can the test name be more descriptive?
- for this and the tests below, can we exhaustively test that the other filing statuses are NOT "X"? These tests would all still pass if we changed the code to just set an X on every filing status.